package.json

```diff
@@ -59,12 +59,12 @@
     "deploy": "gh-pages -d docs",
     "storybook": "start-storybook -p 9009",
     "build-storybook": "build-storybook -o docs",
-    "deploy:storybook": "yarn build-storybook && yarn gh-pages -d docs"
+    "deploy:storybook": "yarn build-storybook && yarn gh-pages -d docs",
+    "prop-types": "^15.5.4"
```

Comment:
This is in the wrong place. `yarn prop-types`?

Reply:
oh, right that is a real dependency. Whoops!

Reply:
Done!